specs/SUMMARY.md

```diff
@@ -9,6 +9,7 @@
 - [Architecture](Architecture.md)
 - [Chain specification](ChainSpec/README.md)
```

Comment:
Shall we remove a link here and delete `ChainSpec/README.md`? So that we don't have to maintain multiple lists consistent.